tools/importer/tfm_importer.json

```diff
@@ -64,12 +68,12 @@
     }
   ],
   "commit_sha": [
-    "fb068d2cb4e89cacf0e9f413075bb4b211f1484f",
-    "5d41a2aeae71b13f2763bea4e55899646291e0eb",
-    "9c1e080e39adc7211c8c2c12cd652da3dc124299",
-    "78ed87028718b1b926d847ff6fc2f91d44e53d6d",
-    "280715f9b74ab29459d81edaf02b39e7a6acb13c",
-    "ea81bf91c90ae23dd9de012bfd7498613be00601",
-    "5342015bb12a486a1c563175a8a7129f0737c925"
+    "11bff3f3cbfbd3e2c284e884d0066531e6b47d7e",
+    "795e6418d0e73841868b351b605659a05c04e1f6",
+    "35938a407133fe0c20c25b6fae2836148d1adfca",
+    "910a402ce6c96b654cb6ae1a5b679e4f856c5419",
+    "d3c610e090282893ba1820c30fc8be3c03195091",
+    "fb35880cecff7205fd19aaaed02b9958b9bfce5a",
+    "be5cfdf18e6e70c9360dea7456c85189a56f277e"
```

Comment:
a while ago i've added an option to specify a help message/description for each hash. Take a look here https://github.com/ARMmbed/mbed-os/blob/master/tools/importer/importer.py#L239

Consider specifying a commit message for each hash - after couple of months when you will be back for new re-import - it will help you a lot.

Reply:
@alzix Ah cool. Thanks. I'll update the json file.